### PR TITLE
fix: filter bot from contributors

### PR DESCRIPTION
**Pull request by Hossein-Mirazimi**
this closes: #1384

### CODE REVIEW

File: components/home/HomeSectionContributors.vue [hunk 1]
```diff
@@ -21,7 +21,7 @@ function onIntersectionObserver ([{ isIntersecting }]) {
 // Fetch on client-side
 onMounted(async () => {
   if (contributors.value.length) return
-  _contributors = await $fetch('https://api.nuxt.com/contributors').then(data => data.slice(0, total * 10).map(c => c.username))
+  _contributors = await $fetch('https://api.nuxt.com/contributors').then(data => data.filter(username => !['codecov-io'].includes(username)).slice(0, total * 10).map(c => c.username))
```
Comment (benjamincanac) [inline components/home/HomeSectionContributors.vue:24]: I guess we can also add `codecov-commenter` here that's showing up on second page:

CleanShot 2023-10-23 at 15 50 13@2x

**Comment by Hossein-Mirazimi** [comment]
I have an idea for improving bot filtering performance. My idea is to split the list into two parts, with the first containing 202 elements (200 plus the bot list's length), and then apply the filter. Afterward, I would slice it again to keep the first 200 elements. I believe this approach would result in better performance.

File: components/home/HomeSectionContributors.vue [hunk 2]
```diff
@@ -21,7 +21,7 @@ function onIntersectionObserver ([{ isIntersecting }]) {
 // Fetch on client-side
 onMounted(async () => {
   if (contributors.value.length) return
-  _contributors = await $fetch('https://api.nuxt.com/contributors').then(data => data.slice(0, total * 10).map(c => c.username))
+  _contributors = await $fetch('https://api.nuxt.com/contributors').then(data => data.filter(username => !['codecov-io', 'codecov-commenter'].includes(username)).slice(0, total * 10).map(c => c.username))
```
Comment (benjamincanac) [inline components/home/HomeSectionContributors.vue:24]: I guess this won't work as the data you're receiving is an object, not an array of strings.

**Comment by Atinux** [comment]
Closing since I pushed a fix on https://api.nuxt.com/contributors

Thank you very much for your help @Hossein-Mirazimi 🙏

**Pull request closed without merging**